app/serializers/user_project_preference_serializer.rb

```diff
@@ -16,6 +16,10 @@ def activity_count
     end
   end
 
+  def activity_count_by_workflow
+    UserSeenSubject.activity_by_workflow(@model.user_id)
```

Comment:
So - this is going to return all the workflows for all projects that a user has contributed too, not the ones for the particular project preference, see `user_project_activity` below. 

I'm also wondering if we can't cache the per workflow method here and use it in the below `user_project_activity` via a local cache? They seem to be doing very similar ops.